utils/sl/components/__init__.py

```diff
@@ -282,7 +282,8 @@ def define_components(reqs):
                                  True)
     reqs.define('argobots',
                 retriever=retriever,
-                commands=['./autogen.sh',
+                commands=['git clean -dxf',
```

Comment:
It's a pity that this got put back.  It's yucky when you are iterating over the building of the stack with some component further down the list, as it means completely defeating the entire utility of `make` and rebuilding this from scratch every time.

Modulo a bug that should be filed upstream, this should be unnecessary.  I would suggest that it's up to higher layers to decide if the workspace should start out clean or if the utility of `make` should be allowed to be enjoyed here.

Reply:
I'll try and remember to put it in next time I'm working on this file.